pom.xml

```diff
@@ -241,19 +241,16 @@ SOFTWARE.
       <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.22.2</version>
-        <dependencies>
-          <dependency>
-            <groupId>org.apache.maven.surefire</groupId>
-            <artifactId>surefire-junit-platform</artifactId>
-            <version>2.22.2</version>
-          </dependency>
-        </dependencies>
+        <version>3.0.0-M5</version>
       </plugin>
       <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-failsafe-plugin</artifactId>
+        <version>3.0.0-M5</version>
         <configuration>
-          <parallel>none</parallel>
+          <forkCount>1</forkCount>
+          <reuseForks>true</reuseForks>
+          <useSystemClassLoader>false</useSystemClassLoader>
```

Comment:
Why do we need such a configuration here?

Reply:
It's to fix these warnings :
```cmd
[WARNING] useSystemClassLoader setting has no effect when not forking
[WARNING] The parameter forkCount should likely not be 0, not forking a JVM for tests reduce test accuracy, ensure to have a <forkCount> >= 1.
```

Reply:
You can submit an issue about these warnings here https://github.com/artipie/ppom/issues and we'll discuss how to solve it

Reply:
Done here : https://github.com/artipie/ppom/issues/82